CampaignConfig/mdc2020_DIOTail.ini

```diff
@@ -20,7 +20,7 @@ stage_type=regular
 output_ancestor_depth=1
 
 [campaign_stage DIOTail_fcl]
-param_overrides = [["--stage ", "DIOTail_fcl"], ["-Oglobal.primary_name=", "DIOtail"],["-Oglobal.stopped=", "Muminus"],["-Oglobal.startmom=", "75"],["-Oglobal.endmom=", "105"],["-Oglobal.njobs=", "100"],["-Oglobal.evts=", "1000"]]
+param_overrides = [["--stage ", "DIOTail_fcl"], ["-Oglobal.primary_name=", "DIOtail"],["-Oglobal.stopped=", "Muminus"],["-Oglobal.startmom=", "75"],["-Oglobal.endmom=", "105"],["-Oglobal.njobs=", "10000"],["-Oglobal.evts=", "2097433"]]
```

Comment:
where does this number of events come from?  It is usually convenient to use a round number in production so we can make quick computations.